### ISSUE TITLE
Plugin options error for `gatsby-remark-images`

### ISSUE THREAD

**Issue by anmol098** [gatsbyjs/gatsby#28217]
## Description

```
error Invalid plugin options for "gatsby-remark-images":

- "tracedSVG" must be a boolean
```

### Steps to reproduce

Having this issue when trying to run npm run develop

How to Make a Minimal Reproduction: https://www.gatsbyjs.org/contributing/how-to-make-a-reproducible-test-case/

### Expected result

It should run the development server
### Actual result

Its crashing unexpectedly 

### Environment

System:
    OS: Windows 10 10.0.19042
    CPU: (8) x64 Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz
  Binaries:
    Node: 12.0.0 - C:\Program Files\nodejs\node.EXE
    npm: 6.9.0 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Python: 3.8.2 - C:\Users\Anmol\AppData\Local\Programs\Python\Python38-32\python.EXE
  Browsers:
    Chrome: 87.0.4280.66
    Edge: Spartan (44.19041.423.0), Chromium (87.0.664.41)
  npmPackages:
    gatsby: ^2.18.7 => 2.27.0
    gatsby-image: ^2.6.0 => 2.6.0
    gatsby-plugin-google-analytics: ^2.1.29 => 2.6.0
    gatsby-plugin-manifest: 2.6.1 => 2.6.1
    gatsby-plugin-netlify: ^2.1.27 => 2.6.0
    gatsby-plugin-offline: ^3.0.27 => 3.5.0
    gatsby-plugin-react-helmet: ^3.1.16 => 3.5.0
    gatsby-plugin-robots-txt: ^1.5.0 => 1.5.3
    gatsby-plugin-sharp: ^2.3.5 => 2.9.0
    gatsby-plugin-sitemap: ^2.2.22 => 2.7.0
    gatsby-plugin-styled-components: ^3.1.14 => 3.5.0
    gatsby-remark-code-titles: ^1.1.0 => 1.1.0
    gatsby-remark-external-links: 0.0.4 => 0.0.4
    gatsby-remark-images: 3.5.1 => 3.5.1
    gatsby-remark-prismjs: ^3.3.25 => 3.8.0
    gatsby-source-filesystem: ^2.1.40 => 2.6.0
    gatsby-transformer-remark: ^2.6.39 => 2.11.0
    gatsby-transformer-sharp: ^2.3.7 => 2.7.0

**Comment by pieh** [comment]
Thanks for reporting the issue, I just confirmed it locally and will be working on a fix for it.

As a workaround you might consider downgrading `gatsby-remark-images` to version `3.3.40` which is last version that doesn't have the validation support and hence wouldn't result in errors like that

**Comment by pieh** [comment]
Fix was published in `gatsby-remark-images@3.7.1`
